nav2_bringup/launch/bringup_multirobot_launch.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import (DeclareLaunchArgument, IncludeLaunchDescription, SetEnvironmentVariable)
+from launch.actions import OpaqueFunction, GroupAction
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration, TextSubstitution
+from nav2_common.launch import ReplaceString
+
+def generate_bringup_multirobot(context):
+    bringup_dir = get_package_share_directory('nav2_bringup')
+    launch_dir = os.path.join(bringup_dir, 'launch')
+    bringup_launch_script = PythonLaunchDescriptionSource(os.path.join(launch_dir, 'bringup_launch.py'))
+
+    slam = LaunchConfiguration('slam')
+    map_yaml_file = LaunchConfiguration('map')
+    use_sim_time = LaunchConfiguration('use_sim_time')
+    params_file = LaunchConfiguration('params_file')
+    autostart = LaunchConfiguration('autostart')
+    use_composition = LaunchConfiguration('use_composition')
+    use_respawn = LaunchConfiguration('use_respawn')
+    log_level = LaunchConfiguration('log_level')
+
+    num_of_robots = context.launch_configurations['number_of_robots']
+    namespace = context.launch_configurations['namespace']
+    bringup_cmd_group = []
+    print(f"number_of_robots = {num_of_robots}")
+
+    for n in range(int(num_of_robots)):
+        new_namespace = [namespace, TextSubstitution(text=str(n))]
+        # Specify the actions
+        bringup_with_ns_cmd_params = IncludeLaunchDescription(
```

Comment:
Where's the substitution for the robot namespace in the params field that's the basis of this work?